nearcore/src/config.rs

```diff
@@ -60,9 +63,19 @@ pub const BLOCK_PRODUCTION_TRACKING_DELAY: u64 = 100;
 /// Expected block production time in ms.
 pub const MIN_BLOCK_PRODUCTION_DELAY: u64 = 600;
 
+/// Mainnet and testnet validators are configured with a different value due to
+/// performance values.
+pub const MAINNET_MIN_BLOCK_PRODUCTION_DELAY: u64 = 1_300;
+pub const TESTNET_MIN_BLOCK_PRODUCTION_DELAY: u64 = 1_000;
```

Comment:
`EXPECTED_EPOCH_LENGTH` depends on `MIN_BLOCK_PRODUCTION_DELAY`. Do we need to do something about that in resetting actual `min_block_production_delay`?

Reply:
Good question. Wasn't aware of `EXPECTED_EPOCH_LENGTH`. It's only used for creating genesis for a new chain. both testnet and mainnet have their genesis already generated, should be fine.